rest-api/src/main/java/gov/cms/qpp/conversion/api/services/StorageService.java

```diff
@@ -16,5 +16,5 @@
 	 * @param inStream The {@link InputStream} to write out to an object in S3.
 	 * @return A {@link CompletableFuture} that will eventually contain the S3 object key.
 	 */
-	CompletableFuture<String> store(String keyName, InputStream inStream);
+	CompletableFuture<String> store(String keyName, InputStream inStream, long size);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document the parameter(s): size [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)